src/main/java/com/direwolf20/buildinggadgets/common/config/InGameConfig.java

```diff
@@ -0,0 +1,243 @@
+package com.direwolf20.buildinggadgets.common.config;
+
+import com.direwolf20.buildinggadgets.common.BuildingGadgets;
+import com.direwolf20.buildinggadgets.common.network.PacketHandler;
+import com.direwolf20.buildinggadgets.common.network.PacketSyncConfig;
+import com.direwolf20.buildinggadgets.common.tools.ReflectionTool;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.gameevent.PlayerEvent.PlayerLoggedInEvent;
+
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Mod.EventBusSubscriber(modid = BuildingGadgets.MODID)
+public class InGameConfig {
```

Comment:
This name triggers me but I'm sure that's just a me issue. Whats the issue with having it as `Config`?

Reply:
The Problem is that we'd have to force forge to reload our config everytime we enter a new game, as it might have been overridden by some Server previously...
Basically I'm using the normal config as a master copy here and then update the InGameConfig as needed

Reply:
The Name triggers me too, but I sadly don't have any ideas how to call it...
(besides SyncedConfig, but that's already how I called the annotation)

Reply:
Does `Config` not make sense? I know we have a class called that already but it would make more sense to move that class to a different name?

Reply:
In my opinion not so much, as it isn't really a Forge Config, but rather an synced Copy of it.
Though, I don't know why I didn't think of that, but we could rename the @SyncedConfig to something like @SyncedValue or @AutoSync and then call the InGameConfig -> SyncedConfig...?
(@AutoSync sounds best in my opnion)

Reply:
👍